Adding HTTPS check in Grafana Annotation Script

kforeverisback opened the pull request:
# Type of PR

- [ ] Documentation changes
- [X] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
HTTPS inclusion in Grafana annotation script

## Validation

- [X] Tested locally with Docker and Grafana with self-signed cert
- [X] Tested on akdc-kind with self-signed cert

## Issues Closed or Referenced

- Closes #602
- References #602

jomalsan commented:
I don't currently have a grafana instance to test the script, but can set one up this afternoon.

In the meantime, can you add a README to this spike directory giving a little context around the annotations that are being added to grafana. Details on where to find the dashboard and panel IDs would be helpful

kforeverisback commented:
@jomalsan I've re-added previous readme and updated that according to our changes. And I've added how to find the Dashboard ID and Panel ID.

I have modified the [akdc-kind](https://github.com/retaildevcrews/akdc-kind/) and ran it locally, and used that as my test bed. I didn't include the grafana setup, but if it's needed it can be included.

jomalsan reviewed (approved):
Thank you for adding the README and the demonstration of running the script! Looks good to me
